src/GitVersionCore.Tests/TestFileSystem.cs

```diff
@@ -1,52 +1,188 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Text;
+
 using GitVersion.Helpers;
 
+using LibGit2Sharp;
+
+using NSubstitute;
+
 public class TestFileSystem : IFileSystem
```

Comment:
Yea, I hoped the abstraction would be useful, but I ended up not using it. Mocking with `static` classes and methods is nigh impossible.